### ISSUE TITLE
Bug: "class" attribute on SVG is incorrectly output as "className"

### ISSUE THREAD

**Issue by matthew-dean** [adamhaile/surplus#36]
It took me a while for my eyes to spot just why the heck my styles weren't getting applied. Turns out my element starts as:
```html
            <svg
                version="1.1"
                class={ $style.callout__quotation }
                x="0px"
                y="0px"
                viewBox="0 0 48 42.1"
            >
                <polygon points="48,0 48,11.6 39.6,19.5 48,19.5 48,42.1 27.5,42.1 27.5,19.7" />
                <polygon points="20.5,0 20.5,11.6 12,19.5 20.5,19.5 20.5,42.1 0,42.1 0,19.7" />
            </svg>
```
and ends up as:
```html
<svg version="1.1" className="_callout__quotation_kxemy_95" x="0px" y="0px" viewBox="0 0 48 42.1"><polygon points="48,0 48,11.6 39.6,19.5 48,19.5 48,42.1 27.5,42.1 27.5,19.7"></polygon><polygon points="20.5,0 20.5,11.6 12,19.5 20.5,19.5 20.5,42.1 0,42.1 0,19.7"></polygon></svg>
```
So, `class` becomes `className`, which of course doesn't apply the CSS class.

**Comment by matthew-dean** [comment]
I was able to workaround this for my Surplus demo tomorrow with:

```jsx
    const svgClass = { class: $style.callout__quotation }
//...
            <svg
                version="1.1"
                {...svgClass}
                x="0px"
                y="0px"
                viewBox="0 0 48 42.1"
            >
```
😄

**Comment by adamhaile** [comment]
Hmm, good catch.  Looks like the HTML-friendly property aliases, which translate class -> className, are firing for SVG too, which they shouldn't.  In fact, they should probably fire in reverse, translating className -> class, since we set attributes, not properties, on SVG nodes.

Have to think about this one.  There's a quick fix which will get almost all the SVG tags but not all.  The ones it would miss would be the tags which are common to both SVG and HTML, like <a>, <title>, etc.  To get them, I have to add a little more context to the translation layer.  I'd like to make a 100% fix for this.

**Comment by matthew-dean** [comment]
@adamhaile No worries, there's no hurry, I can use the workaround for now.

**Comment by matthew-dean** [comment]
@adamhaile Ah, that's also cool. I'll save that for next time!

**Comment by adamhaile** [comment]
Should be fixed with 5ceff5b29e5dcff989c425d4a0e8b2e589225686 .

**Comment by adamhaile** [comment]
Unit tests are passing now, so closing this issue.  Let me know if you have any other problems with it!
